### ISSUE TITLE
Add Warning for double import through public import of imported package

### ISSUE THREAD

**Issue by Frotty** [peq/WurstScript#288]
````
package A

package B
    import public A

package C
    import A <-- mark as excessive
    import B
````

**Comment by Frotty** [comment]
:D

**Comment by Cokemonkey11** [comment]
Context? Did you close this because it's done or because it's unnecessary

**Comment by peq** [comment]
Was fixed with d022b4a819ca0b45ee32d8f3d9d2d8676991dd70
